source/configure/configuration-settings.rst

```diff
@@ -4279,6 +4278,37 @@ Token
 
 Generate a token to validate incoming requests from AWS SNS by selecting ``Regenerate``.
 
+Calls
+~~~~~
+
+Access the following configuration settings in the System Console by going to **Plugins > Calls**.
+
+Enable Plugin
+^^^^^^^^^^^^^
+
+|all-plans| |self-hosted| |cloud|
+
+**True**: Enables the calls plugin on your Mattermost workspace.
+
+**False**: Disables the calls plugin on your Mattermost workspace.
+
```

Comment:
In the section below I've provided the names of the options but not the details as this is indicated in the UI. @cwarnermm, I'm open to changing this if the assumption is that users may read these docs in isolation. My assumption is that they'd read them while editing the settings and wouldn't need a duplicated description.

Reply:
Agreed! I imagine that some decision-makers may review our config settings to get a sense of how configurable the product is in general, but that our primary use case here is someone actively looking to enable and use calls functionality in Mattermost Channels.